print/print-apps/oereb/glossar.jrxml

```diff
@@ -132,7 +132,7 @@
 				<textElement>
 					<font fontName="Cadastra" size="6.5"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$P{CreationDate}]]></textFieldExpression>
+				<textFieldExpression><![CDATA[$P{CreationDate}.toString().substring(8,10)+'.'+$P{CreationDate}.toString().substring(5,7)+'.'+$P{CreationDate}.toString().substring(0,4)+' '+$P{CreationDate}.toString().substring(11,13)+':'+$P{CreationDate}.toString().substring(14,16)+':'+$P{CreationDate}.toString().substring(17,19)]]></textFieldExpression>
```

Comment:
What's this incomprehensible thing...

Shouldn't we use something like:
```
new java.text.SimpleDateFormat("HH mm ss dd MM yyyy").format(new java.text.SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss").parse($P{CreationDate}))
```

To be more understandable ...

Reply:
Or better:
```
java.time.format.DateTimeFormatter.ofPattern("HH mm ss dd MM yyyy").format(java.time.LocalDateTime.parse($P{CreationDate}))
```

Reply:
Thanks, I'm absolutely not familiar with the java functionnality in Jasper, so your suggestion is perfect for me. I'll put it in and push an update 👍